Sources/App/Controllers/PackageController+routes.swift

```diff
@@ -87,6 +87,7 @@ struct PackageController {
         var packageName: String
         var docArchives: [String]
         var latest: Version.Kind?
+        var kind: Version.Kind
```

Comment:
It doesn't seem necessary to make this a property - we already have `reference` on this type and can get `reference.versionKind` from it (the same that's assigned into the value). Is there a reason we need this I'm missing?

I've removed the property and simply set `DocumentationPageProcessor(... referenceKind: documentation.reference.versionKind, ...)` and that seems to work fine?